scrapers/MindGeekAPI.py

```diff
@@ -0,0 +1,208 @@
+import difflib
+import json
+import os
+import re
+import sys
+from configparser import ConfigParser, NoSectionError
+from datetime import datetime
+from urllib.parse import urlparse
+
+import requests
+
+# Not necessary but why not ?
+user_agent = 'Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:79.0) Gecko/20100101 Firefox/79.0'
+
+# Set Variable
+set_ratio = 0.8
+set_fileurl = "MindGeekAPI.ini"
+
+
+def scraping_url(url):
+    id, instance_token = get_info(url)
+    headers = {
+        'Instance': instance_token,
+        'User-Agent': user_agent
+    }
+    return id, headers
+
+
+def get_info(url):
+    todaystr = datetime.today().strftime('%Y-%m-%d')
+    today = datetime.strptime(todaystr, '%Y-%m-%d')
+    token, id = check_config(url, today)
+    if not token:
+        try:
+            r = requests.get(url,timeout=(3, 5))
+        except requests.Timeout:
+            print("Request Timeout", file=sys.stderr)
+            exit(1)
+        id = re.search('(.+releaseId\":\")(.+?)(\".+)',
+                       r.text, re.IGNORECASE).group(2)
+        token = re.search('(.+jwt\":\")(.+?)(\".+)',
+                          r.text, re.IGNORECASE).group(2)
```

Comment:
if a request is made for a site that has not the above regexes (eg not  in the supported list ) then no match is made and so no group will be available
`AttributeError: 'NoneType' object has no attribute 'group'`
those should be `tried` probably